backend/routes/PostReaction/CommentHelperFunc.py

```
@@ -0,0 +1,26 @@
+
+# ----------------------------------------
+# Helper Functions
+# ----------------------------------------
+
+def get_comment_by_id(db: Session, comment_id: int) -> Comment:
+    comment = db.query(Comment).filter(Comment.id == comment_id).first()
+    if not comment:
+        raise HTTPException(status_code=404, detail="Comment not found.")
+    return comment
+
+def authorize_comment_deletion(comment: Comment, post: Post, user_id: int):
```

Ruff (0.8.2)

Missing return type annotation for public function `authorize_comment_deletion`

Add return type annotation: `None`

(ANN201)